docs/components/concepts/clusters.md

```diff
@@ -0,0 +1,44 @@
+---
+id: clusters
+title: "Clusters"
+description: "Learn more about the clusters available in your Camunda Platform 8 plan."
+---
+
+A cluster is a provided group of nodes that run Camunda Platform 8. By default, Camunda Platform 8 clusters are production-ready.
+
+Professional and Enterprise plan customers can create as many production or development clusters as they want.
+
+The following table shows each plan and available type or size of cluster:
+
+|              | Development | Production - S | Production - M | Production - L |
+| ------------ | ----------- | -------------- | -------------- | -------------- |
+| Free         | \-          | \-             | \-             | \-             |
```

Comment:
Should Free Trial technically come before the Free Plan?

Reply:
I like this. Like the order you could experience the plans.